frontend/src/components/MakerForm/MakerForm.tsx

```diff
@@ -77,6 +84,7 @@ const MakerForm = ({
 
   const [openDialogs, setOpenDialogs] = useState<boolean>(false);
   const [submittingRequest, setSubmittingRequest] = useState<boolean>(false);
+  const [amountRangeEnabled, setAmountRangeEnabled] = useState<boolean>(false);
```

Comment:
If we make it true by default `useState<boolean>(true)`, we can remove the extra `setAmountRangeEnabled()` on the handling of the AdvancedOptions switch. This way amountRangeEnabled persists when the user disables/enables advanced options.

Reply:
Resolved in the last commit 375eb495016decfc2bffd61172e8ab50fce5dc8a.